### PR TITLE
Updated Enterprise Quickstart with updates from Cloud quickstart

### PR DESCRIPTION
**Pull request by jwitz**
This PR synthesizes content from the Enterprise quickstart with improvements that were made to the Cloud quickstart guide last month. In general, the content hasn't changed - I updated the doc in the following ways:

- Reorganized steps to be more cohesive, as well as to follow a more natural pattern for installing certain features
- Linked to some new docs and improved wording to match our style guide

This update doesn't include screen shots from Airflow 2.0, but definitely can if we decide it's not disruptive to add those now.

### CODE REVIEW

File: enterprise/next/01_get-started/01_quickstart.md [hunk 1]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
-Astronomer Cloud:
+## Step 1: Start with Astronomer Cloud
 
-- Runs in Astronomer's VPC and uses the public internet, whereas Astronomer Enterprise will run in *your* environment according to your own security settings
-- Is billed by usage, whereas Enterprise is billed through an annual license
-- Does not give users access to the same monitoring stack
+If you want to get a sense of Astronomer Enterprise without going through the installation process, try out Cloud by [starting a 14-Day Trial](/trial/). At its core, Astronomer Cloud is a large-scale deployment of Astronomer Enterprise that is fully managed by our team. Cloud provides parallel Airflow functionality to Enterprise with just a few key differences:
 
-If you want to get a sense of Astronomer Enterprise without going through the installation process, try out Cloud by [starting a 14-Day Trial](/trial/).
+- Astronomer Cloud runs in Astronomer's VPC and uses the public internet, whereas Astronomer Enterprise runs in your own environment according to your own security settings
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:17]: Should these have a period at the end?

(Love the language/structure changes here by the way)

File: enterprise/next/01_get-started/01_quickstart.md [hunk 2]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
+- Astronomer Cloud is billed by usage, whereas Enterprise is billed through an annual license
+- Astronomer Cloud doesn't not give users access to the same monitoring stack you have in Astronomer Enterprise
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:19]: Typo on "doesn't not". Maybe also specify that that monitoring stack includes Grafana and Kibana?

File: enterprise/next/01_get-started/01_quickstart.md [hunk 3]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
-
-Flags:
-  -h, --help   help for astro
-
-Use "astro [command] --help" for more information about a command.
-```
+## Step 3: Install Astronomer Enterprise
 
-### Create a project
+If you are in charge of setting up Astronomer for your org, follow one of our Enterprise installation guides to get Astronomer running on your Kubernetes Cluster.
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:56]: "For your team" instead of "org"?

File: enterprise/next/01_get-started/01_quickstart.md [hunk 4]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
 
-Your first step is to create a project to work from that lives in on your local machine.
+We have documentation for deploying Astronomer on:
+- AWS EKS ([Manually](/docs/enterprise/stable/install/aws/install-aws-standard/) or through [Terraform](/docs/enterprise/stable/install/aws/install-aws-terraform/))
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:59]: Should we call it "Manual" or "Standard"?

File: enterprise/next/01_get-started/01_quickstart.md [hunk 5]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
 
-## 5. Create an Airflow deployment
+To help you get started, your initialized project includes an `example-dag` in `/dags`. This DAG simply prints today's date, but it'll give you a chance to become familiar with how to deploy on Astronomer.
 
-Run through the prompts in the UI to create your first workspace and Airflow deployment.
+If you'd like to deploy some more functional DAGs, upload your own or check out [example DAGs we've open sourced](https://github.com/airflow-plugins/example-dags).
 
-![ETL Workspace](https://assets2.astronomer.io/main/docs/enterprise_quickstart/etl_workspace.png)
 
+## Step 5: Identify Your Base Domain and Log In
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:107]: "Log into the Astronomer UI"? Might be too wordy, I don't feel strongly (at first I read this header and thought section was for CLI)

File: enterprise/next/01_get-started/01_quickstart.md [hunk 6]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
-![Create Deployment](https://assets2.astronomer.io/main/docs/enterprise_quickstart/create_deployment.png)
 
-Generally, it'll take 1-2 minutes for each environment to spin up.
+> **Note:** If you are not the first person to log in, you will need an email invite to the platform.
 
-## 6. Deploy some DAGs
+## Step 6: Create a Workspace
 
-Now there's an Airflow deployment running on your Astronomer installation, you're ready to push code!
+If you're the first person to log in to the Astronomer UI, click **New Workspace** to create a Workspace.
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:118]: Lol I never know which one should be used where, but should this be "log into"?

File: enterprise/next/01_get-started/01_quickstart.md [hunk 7]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
+You can think of Workspaces the same way you'd think of teams - a space that specific user groups have access to with varying levels of permissions. From within a Workspace you can create one or more Airflow Deployments, each of which hosts a collection of DAGs.
 
-Navigate to the `hello-astro` project created before:
+For more information, read [Manage Workspaces and Deployments](/docs/enterprise/stable/deploy/manage-workspaces/).
 
+## Step 7: Authenticate from the CLI
+
+You can authenticate to Astronomer using the following command:
+```sh
+astro auth login BASEDOMAIN
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:130]: Add $

File: enterprise/next/01_get-started/01_quickstart.md [hunk 8]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
-INFO[0016] [2/3] [webserver]: Starting
-INFO[0016] [3/3] [webserver]: Started
-Airflow Webserver: http://localhost:8080
-Postgres Database: localhost:5432/postgres
-The default credentials are admin:admin
+You'll be prompted to authenticate and select the Workspace that you want to log in to.
 
-```
-Navigate to `http://localhost:8080` in the browser to get to Airflow.
+The first user to log in will become the Admin. Additional users can be added from the Astronomer UI. For more information on Astronomer's users and permissions model, read [Manage Users on Astronomer Enterprise](/docs/enterprise/stable/manage-astronomer/manage-platform-users/).
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:135]: Specify "System Admin"

File: enterprise/next/01_get-started/01_quickstart.md [hunk 9]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
-5c858070b896: Mounted from elementary-rocket-7360/airflow
-95df7c26d7c5: Mounted from elementary-rocket-7360/airflow
-77cae8ab23bf: Mounted from elementary-rocket-7360/airflow
-deploy-1: digest: sha256:9e69b7b81efc95c3f024f7656646b523f5b611c1954d52be6dca203c888fc842 size: 3230
-Untagged: registry.democluster.astronomer-trials.com/dynamical-flare-3582/airflow:deploy-1
-Untagged: registry.democluster.astronomer-trials.com/dynamical-flare-3582/airflow@sha256:9e69b7b81efc95c3f024f7656646b523f5b611c1954d52be6dca203c888fc842
-Deploy succeeded!
+For production Deployments, you'll likely need to reconfigure the Deployment to fit the needs of your organization. For more information on configuring Deployments, read [Configure an Airflow Deployment on Astronomer](https://www.astronomer.io/docs/enterprise/stable/deploy/configure-deployment).
+
+## Step 9: Deploy a DAG
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:152]: This is all local stuff - I'm wondering if we pull it back up to the top of the doc and have this section *before* the install. What do you think?

File: enterprise/next/01_get-started/01_quickstart.md [hunk 10]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
+```
 
-Once the `example_dag` has been turned on, navigate to the `Metrics` tab in the Astronomer UI to see the metrics flow in in real time.
+4. Confirm the deploy was successful by going to your local Airflow instance at `http://localhost:8080`. You should be able to view `example-dag` in the Airflow UI. You should also be able to turn on `example-dag` and see task runs begin to accumulate in the **Recent Tasks** column.
 
-![Metrics](https://assets2.astronomer.io/main/docs/enterprise_quickstart/metrics.png)
+## Step 10: Access Metrics
 
-## 9. Use the Admin tools
+Once you've turned on the `example-dag` in the Airflow UI, go to the **Metrics** tab in the Astronomer UI to see the Grafana metrics for your Deployment in real time. For a full breakdown of the Grafana dashboard, read [Metrics in Astronomer Enterprise](https://www.astronomer.io/docs/enterprise/stable/monitor/grafana-metrics).
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:185]: Specify this is only available to SysAdmins - unless you're talking about Metrics tab? Totally different than Grafana, so would rephrase to remove that reference in this case

File: enterprise/next/01_get-started/01_quickstart.md [hunk 11]
```diff
@@ -4,252 +4,197 @@ navTitle: "Quickstart"
...
-## 10. Start inviting Users
+## What's Next?
 
-Now that you've run through all of the functionality on Astronomer enterprise, you're ready to start inviting users!
+Once you've successfully installed your Astronomer platform, we recommend doing the following:
 
-To change the authentication system, naviate to our [Auth Systems](/docs/enterprise/stable/manage-astronomer/integrate-auth-system/) doc.
+* [Invite new users to Astronomer](https://www.astronomer.io/docs/enterprise/stable/manage-astronomer/manage-platform-users)
+* [Manage permissions](https://www.astronomer.io/docs/enterprise/stable/manage-astronomer/workspace-permissions) for your new users
+* Integrate an [Auth System](/docs/enterprise/stable/manage-astronomer/integrate-auth-system/)
```
Comment (paolaperaza) [inline enterprise/next/01_get-started/01_quickstart.md:200]: Maybe add "Set up CI/CD"?

**Review by paolaperaza — CHANGES_REQUESTED** [review]
@jwitz This is great! I'm glad we're taking a chance to clean up the flow here. I think this guide is ultimately best suited for individual data engineers/users of Airflow whose team already installed the platform, and I think this structure suits that well.

I made a few comments on copy + one about moving up local instructions. Lemme know what you think and I'll give it another look.

**Comment by paolaperaza** [comment]
Also - yes to screenshots of 2.0 if possible!

File: enterprise/next/01_get-started/01_quickstart.md (hunk 1 above)
Comment (jwitz) [reply enterprise/next/01_get-started/01_quickstart.md:17]: Yep, going forward I'll use the style point of using punctuation:
- after every bullet point that is a sentence. 
- after every bullet point that completes the introductory stem( as these bullets do).

File: enterprise/next/01_get-started/01_quickstart.md (hunk 3 above)
Comment (jwitz) [reply enterprise/next/01_get-started/01_quickstart.md:56]: Nice catch, that's a weird leftover from the old version.

File: enterprise/next/01_get-started/01_quickstart.md (hunk 4 above)
Comment (jwitz) [reply enterprise/next/01_get-started/01_quickstart.md:59]: Since "Standard" isn't an AWS-specific term, I think it's more appropriate to use manual here. Though it differs from what's in the side nav, "Manual" makes sense as it better differentiates this install from a Terraform install (in a context where someone still might be deciding on which install to use).

File: enterprise/next/01_get-started/01_quickstart.md (hunk 5 above)
Comment (jwitz) [reply enterprise/next/01_get-started/01_quickstart.md:107]: How about "Go to Your Base Domain and Log In"? Between that and the screen shot, it think the risk of getting lost here is minimal.

File: enterprise/next/01_get-started/01_quickstart.md (hunk 6 above)
Comment (jwitz) [reply enterprise/next/01_get-started/01_quickstart.md:118]: Giving the last word to Googs on this one:
Screen Shot 2021-01-07 at 8 52 37 AM

The way I see it, you always "log in" but you never just "log" (at least int the context of . So when we decide whether to write "X in" or "X into", I think about whether X is the entire word or not. Kind of a hacky explanation, but that's how it makes sense in my head. 

(Also that's how it looks in our UI, so definitely easier for us to be consistent there than vice versa)

File: enterprise/next/01_get-started/01_quickstart.md (hunk 9 above)
Comment (jwitz) [reply enterprise/next/01_get-started/01_quickstart.md:152]: Configuring a Deployment or step 9? I'm confused about how either of these can be completed outside of the install, as they require astro CLI and a Deployment.

File: enterprise/next/01_get-started/01_quickstart.md (hunk 10 above)
Comment (jwitz) [reply enterprise/next/01_get-started/01_quickstart.md:185]: Ah got it. Is there any other info we should mention here about the Metrics tab?

**Comment by jwitz** [comment]
Thanks for looking at this @paolaperaza ! Left some minor comments and one question, but otherwise was able to resolve everything here.

File: enterprise/next/01_get-started/01_quickstart.md (hunk 1 above)
Comment (paolaperaza) [reply enterprise/next/01_get-started/01_quickstart.md]: Perfect, yea that sounds like what we agreed upon!

File: enterprise/next/01_get-started/01_quickstart.md (hunk 3 above)
Comment (paolaperaza) [reply enterprise/next/01_get-started/01_quickstart.md]: Np!

File: enterprise/next/01_get-started/01_quickstart.md (hunk 9 above)
Comment (jwitz) [reply enterprise/next/01_get-started/01_quickstart.md:152]: Per our conversation on Zoom, we'll keep these steps in for now. But down the line, it would be good to reconsider what to do with this quickstart guide - anyone that's already acquired an Enterprise license is on a journey with Astronomer that is very much not a quick start.

File: enterprise/next/01_get-started/01_quickstart.md (hunk 4 above)
Comment (jwitz) [reply enterprise/next/01_get-started/01_quickstart.md:59]: Yeah I like that!

**Comment by jwitz** [comment]
Suggested changes look good @paolaperaza. Added them in a batch and pushed everything back to v0.16 only (the earliest version anyone would be starting).

Assuming we're tabling discussion on the CLI/ Quickstart overlap, this should be good to merge.

**Review by paolaperaza — APPROVED** [review]
Alright @jwitz - I just went in here and made a few tiny changes and applied them to both `/next` and `v0.16.

Resolved my previous comments as well to make sure we didn't miss anything. I'm going to go ahead and merge! Depending on how the bulleted list formatting shows up, we can continue to iterate.
